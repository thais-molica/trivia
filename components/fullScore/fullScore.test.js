import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";
import FullScore from "./index";

it("FullScore renders correctly", () => {
  const tree = renderer.create(<FullScore correct={1}  error={1} />).toJSON();
  expect(tree).toMatchSnapshot();
});
