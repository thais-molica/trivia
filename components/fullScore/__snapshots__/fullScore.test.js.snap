// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FullScore renders correctly 1`] = `
<div
  className="jsx-948137128"
>
  <span
    className="jsx-948137128"
  >
    <strong
      className="jsx-948137128"
    >
      1
    </strong>
     correct
  </span>
  <span
    className="jsx-948137128"
  >
    <strong
      className="jsx-948137128"
    >
      1
    </strong>
     incorrect
  </span>
</div>
`;
